api/app/views/mno_enterprise/auth/sessions/new.html.haml

```diff
@@ -1,6 +1,9 @@
 .registration
   .container
     .row
+      - if params[:session_timeout].present?
+        .alert.alert-warning.text-center
+          You are logged off due to inactivity. Please log in again
```

Comment:
You should add a translation in this file:
https://github.com/maestrano/mno-enterprise/blob/master/core/config/locales/views/auth/sessions/en.yml